mlflow/onnx.py

```diff
@@ -130,6 +131,17 @@ def save_model(
                                      This uses GPU preferentially over CPU.
                                      See onnxruntime API for further descriptions:
                                      https://onnxruntime.ai/docs/execution-providers/
+    :param onnx_session_options: Dictionary of options to be passed to onnxruntime.InferenceSession.
+                                 For example:
+                                 ``{
+                                 'graph_optimization_level': 99,
+                                 'intra_op_num_threads': 1,
+                                 'inter_op_num_threads': 1,
+                                 'execution_mode': 'sequential'
+                                 }``
+                                 'execution_mode' can be set ot 'sequential' or 'parallel'.
```

Comment:
Pls also add the doc in "log_model" method

Reply:
It has been added